packages/commonwealth/client/scripts/models/AddressInfo.ts

```diff
@@ -4,45 +4,40 @@ import app from 'state';
 import Account from './Account';
 
 class AddressInfo extends Account {
-  public readonly id: number;
-  public readonly profileId: number;
+  public readonly userId: number;
 
   constructor({
+    userId,
     id,
     address,
     communityId,
     walletId,
     walletSsoSource,
     ghostAddress,
-    profileId,
     lastActive,
   }: {
-    id: number | null | undefined;
+    userId: number;
+    id: number;
```

Comment:
shouldn't we set `this.id`, if we are keeping its type?

Reply:
this was redundant since the base class has addressId